src/internal/interpreter/execution_engine.rs

```diff
@@ -42,17 +42,49 @@ impl ExecutionEngine for Interpreter {
                     self.stack.pop();
                 } // We have the braces around this call to make the Rust compiler happy
 
-                Instruction::Add(ref dst_reg, ref reg_1, ref reg_2) => self.add(dst_reg, reg_1, reg_2),
-                Instruction::Sub(ref dst_reg, ref reg_1, ref reg_2) => self.sub(dst_reg, reg_1, reg_2),
-                Instruction::Mul(ref dst_reg, ref reg_1, ref reg_2) => self.mul(dst_reg, reg_1, reg_2),
-                Instruction::Div(ref dst_reg, ref reg_1, ref reg_2) => self.div(dst_reg, reg_1, reg_2),
+                Instruction::Add(ref dst_reg, ref reg_1, ref reg_2) => {
```

Comment:
These do not need braces

Reply:
IDE automatically reformatted this to satisfy a line length limit